cmd/server/app/serve.go

```diff
@@ -87,11 +87,19 @@ var serveCmd = &cobra.Command{
 		}
 
 		// Identity
-		jwksUrl, err := cfg.Identity.Server.Path("realms/stacklok/protocol/openid-connect/certs")
+		// TODO: cfg.Identity.Server.IssuerUrl _should_ be a URL to an issuer that has an
```

Comment:
I'm not sure this comment is accurate. Maybe it's better to rename the `issuerUrl` to `hostnameUrl`. We intentionally use the hostname for deriving the user deletion URL as well, which is also Keycloak specific.

Reply:
Ugh... so, in an ideal world, the URL that Keycloak includes in the `iss` claim would be reachable by the `minder` server and then we could use the standard OpenID JWKS discovery.

Unfortunately, the current docker-compose setup produces an `iss` of `http://localhost:8081/realms/stacklok`, but the `minder` container can only reach Keycloak via `keycloak:8080`.  This isn't a problem in staging or production, where Minder _can_ reach the external interface, and we just _happen_ to fetch some keys from an entirely different URL and use them to validate the JWT.

Let me think on this, and see if I can come up with a better way to handle this.

And yes, we should rename `issuer_url` to `identity_host` or `identity_url`, but I didn't want to gratuitously break existing configuration (including our own).

That also reminds me that I need to set these fields in staging and production _before_ we push this code there...